### ISSUE TITLE
Upgrade Microsoft.Extensions.Logging or use abstraction.

### ISSUE THREAD

**Issue by niemyjski** [serilog/serilog-extensions-logging#171]
I'm getting a type load exception with 3.0.1 and I have no direct dependencies on `Microsoft.Extensions.Logging` The only dependency I can see is coming from this library. Currently only .NET Core 3.1 is supported (https://dotnet.microsoft.com/platform/support/policy/dotnet-core) would it be possible to bump deps with a new release?

```
System.TypeLoadException: Could not load type 'Microsoft.Extensions.Logging.Abstractions.Internal.NullScope' from assembly 'Microsoft.Extensions.Logging.Abstractions, Version=3.1.3.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'.
  at at Microsoft.Extensions.Logging.Logger.BeginScope[TState](TState state)
  at at Microsoft.Extensions.Logging.Logger`1.Microsoft.Extensions.Logging.ILogger.BeginScope[TState](TState state)
```

**Comment by nblumhardt** [comment]
Hi Blake, thanks for the note, which TFM are you targeting?

**Comment by Numpsy** [comment]
Same issue as #158?

**Comment by skomis-mm** [comment]
Yes, the same issue  #158 
@niemyjski , it means the binary version of `Microsoft.Extensions.Logging` is different (<3.0.0) from `.Abstractions` version (3.1.3).
Why it happens:
`Serilog.Extensions.Logging` -> `Microsoft.Extensions.Logging (2.0.0)` -> `Microsoft.Extensions.Logging.Abstractions (2.0.0)` 
`<LibraryProject>` -> `Microsoft.Extensions.Logging.Abstractions (3.1.3)`

If you not add reference for `Microsoft.Extensions.Logging` package, the result of nuget dependency resolution will be:
`Microsoft.Extensions.Logging (2.0.0)`
`Microsoft.Extensions.Logging.Abstractions (3.1.3)`

I would suggest in this case if your library targets `Abstractions` package it is better for long term to target the least version possible unless you need specific API from higher version. Or add reference to `Microsoft.Extensions.Logging (3.1.3)` package to the end project

**Comment by niemyjski** [comment]
Yep, same issue.

**Comment by niemyjski** [comment]
Any updates on this?

**Comment by Boditha** [comment]
Any update on this?

**Comment by abakumov-v** [comment]
@skomis-mm Hi!

Thanks, your suggestion works fine for me 👍

**Comment by sungam3r** [comment]
@nblumhardt close?

**Comment by nblumhardt** [comment]
Thanks @sungam3r 👍 fixed in #223
